cmd/sops/subcommand/updatekeys/updatekeys.go

```diff
@@ -23,30 +22,34 @@ type Opts struct {
 }
 
 // UpdateKeys update the keys for a given file
-func UpdateKeys(opts Opts) error {
-	path, err := filepath.Abs(opts.InputPath)
-	if err != nil {
-		return err
-	}
-	info, err := os.Stat(path)
-	if err != nil {
-		return err
-	}
-	if info.IsDir() {
-		return fmt.Errorf("can't operate on a directory")
+func UpdateKeys(inputPaths []string, opts Opts) error {
+	for _, inputPath := range inputPaths {
```

Comment:
This changes a publicly exposed function breaking backwards compatibility. To avoid this, the loop should for example be dealt with in `main.go`, or a new function should be introduced.

Reply:
Adjusted.